docs/components/typography.md

```diff
@@ -0,0 +1,80 @@
+<script setup>
+import { SwText, FONT_FAMILY, FONT_WEIGHTS } from '@swimm/ui';
+</script>
+<style scoped>   
+.theme-default-content h1,
+.theme-default-content h2,
+.theme-default-content h3,
+.theme-default-content h4,
+.theme-default-content h5,
+.theme-default-content h6 {
+    margin: 16px 0;
+    padding: 0;
+    border: none;
+}
+</style>
+
+# Typography
+
+Typography can get props that will change its appearance.
+
+- **component** - the component that will be displayed `div`/`span`/`h1`/`h2`/`h3`/`h4`/`h5`/`h6`
+- **variant** - (default: `VARIANTS.BODY_L`)
+- **family** - the font family that will be used for the font
+- **weight** - the font weight that will be used for the font
+
+### Basic
+
+::: demo
+ <sw-text variant="system-headline">system-headline</sw-text>
```

Comment:
Just wondering, why don't we use the Pascal case convention here?

Reply:
Done

Reply:
@yhamry it wasn't done eventually 😨 hahah